server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bsp/managers/BazelToolchainManager.kt

```diff
@@ -38,7 +39,7 @@ class BazelToolchainManager(private val bazelRunner: BazelRunner, private val fe
           .waitAndGetResult(cancelChecker)
           .isSuccess
       }
-    return if (androidToolchainExists) androidToolchain else NATIVE_ANDROID_TOOLCHAIN
+    return listOf(if (androidToolchainExists) androidToolchain else NATIVE_ANDROID_TOOLCHAIN).filterNotNull()
```

Comment:
```suggestion
    return listOf(if (androidToolchainExists) androidToolchain else NATIVE_ANDROID_TOOLCHAIN)
```
as far as I can see they are not nullable